authentication/backends.py

```diff
@@ -1,10 +1,8 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
 
-"""Configure JWT Here"""
+from rest_framework import authentication, exceptions
 
+"""Configure JWT Here"""
+class JWTAuthentication(authentication.BaseAuthentication):
+    
+    authentication_header_prefix = 'Bearer'
```

Comment:
Implementation is still lacking?

Reply:
Thank you for the review am going to work and fix this ASAP

Reply:
Added more implementation into this JWTAuthentication class